### ISSUE TITLE
[pl.lineage_drivers] cycles the first few genes when n_genes is larger than 8

### ISSUE THREAD

**Issue by mzhibo** [theislab/cellrank#542]
when plotting the top lineage drivers with n_genes >8, it repeats the topfew genes. The number of genes cycled varies when the n_genes and ncols change.

```python
cr.pl.lineage_drivers(adata, lineage="0", n_genes=10)# cycles the top 5 genes
cr.pl.lineage_drivers(adata, lineage="0", n_genes=10, ncols=5)# cycles the top 5 genes
cr.pl.lineage_drivers(adata, lineage="0", n_genes=20, ncols=5)# cycles the top 7 genes
cr.pl.lineage_drivers(adata, lineage="0", n_genes=20)# cycles the top 8 genes
```

#### Versions:

> Running CellRank 1.2.0, on 2021-03-23 09:16.

**Comment by michalk8** [comment]
Thanks for catching this @mzhibo , will be fixed in #543 and merged into the `dev` branch.

**Comment by michalk8** [comment]
closed via #543
